crates/node/src/interactive_cli/peers.rs

```diff
@@ -15,25 +14,26 @@ pub struct PeersCommand {
 }
 
 impl PeersCommand {
-    pub async fn run(self, node: &Node) -> EyreResult<()> {
+    pub async fn run(self, node_client: &NodeClient) -> EyreResult<()> {
         let ind = ">>".blue();
-        println!(
-            "{ind} Peers (General): {:#?}",
-            node.network_client.peer_count().await.cyan()
-        );
 
         let context_id = self
             .context
-            .map(|context| node.ctx_manager.resolve_alias(context, None))
+            .map(|context| node_client.resolve_alias(context, None))
             .transpose()?
             .flatten();
 
+        println!(
+            "{ind} Peers (General): {:#?}",
+            node_client.get_peers_count(context_id).await.cyan()
+        );
+
```

Comment:
this will print the number of peers connected for a context when the context is provided

`General` is for ALL nodes connected to the node, regardless of which context they're connected on

let's revert and just change the relevant bits in place without changing the structure